lib/bolt/transport/api.rb

```diff
@@ -2,26 +2,213 @@
 
 require 'bolt/transport/api/connection'
 
-# A copy of the orchestrator transport which uses the api connection class
-# in order to bypass calling 'start_plan'
 module Bolt
   module Transport
-    class Api < Orch
-      def initialize(*args)
-        super
+    class Api
```

Comment:
I think we should move this out of the `Bolt::Transport` namespace, since it's not a transport in the typical sense. Maybe something under `plan_executor/executor`, since it's really just a part of the executor.

Reply:
`PlanExecutor::OrchClient`?

Reply:
I guess to me it seems like, even though it's only used by the executor it's not logically part of the executor - it's really the interface between the executor and orchestrator. I don't feel strongly about it though if you'd rather I make it a class under  the executor, `PlanExecutor::Executor::OrchClient` or somesuch.